.github/workflows/release-packages.yaml

```diff
@@ -0,0 +1,86 @@
+on:
+  release:
+    types: [created]
+
+name: Upload additional release assets
+jobs:
+  ubuntu-20_04-package:
+    name: Generate ubuntu-20.04 debian package
+    runs-on: ubuntu-20.04
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Should this be `RELEASE_CI_ACCESS_TOKEN`?

Reply:
Got it.

Reply:
https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret for those not familiar